addons/website_sale/views/templates.xml

```diff
@@ -53,6 +53,162 @@
         </head>
     </template>
 
+    <template
+        id="template_footer_website_sale"
+        name="eCommerce Footer"
+        inherit_id="website.layout"
+        active="False"
+    >
+        <div id="footer" position="replace">
+            <div
+                t-if="not no_footer"
+                id="footer"
+                class="oe_structure oe_structure_solo"
+                t-ignore="true"
+            >
+                <section class="s_text_block pt48 pb32" data-snippet="s_text_block" data-name="Text">
+                    <div class="container s_allow_columns">
+                        <div class="row">
+                            <!-- Column 1: Products -->
+                            <div class="col-lg-3 o_colored_level mb-4 mb-lg-0">
```

Comment:
The text block snippet is not supposed to have non editable margin on its columns... unless really necessary, use what the editor can do on the column (see what happens when you increase a column height in the editor).